edalize/symbiflow.py

```diff
@@ -0,0 +1,160 @@
+import logging
+import os.path
+import platform
+import re
+import subprocess
+
+from edalize.edatool import Edatool
+from edalize.yosys import Yosys
+from importlib import import_module
+
+logger = logging.getLogger(__name__)
+
+""" Symbiflow backtend
+
+A core (usually the system core) can add the following files:
+
+- Standard design sources (Verilog only)
+
+- Constraints: unmanaged constraints with file_type SDC, pin_constraints with file_type PCF and placement constraints with file_type xdc
+
+"""
+
+
+class Symbiflow(Edatool):
+
+    argtypes = ["vlogdefine", "vlogparam", "generic"]
+
+    @classmethod
+    def get_doc(cls, api_ver):
+        if api_ver == 0:
+            symbiflow_help = {
+                "members": [
+                    {
+                        "name": "package",
+                        "type": "String",
+                        "desc": "FPGA chip package (e.g. clg400-1)",
+                    },
+                    {
+                        "name": "part",
+                        "type": "String",
+                        "desc": "FPGA part type (e.g. xc7a50t)",
+                    },
+                    {
+                        "name": "builddir",
```

Comment:
I would prefer to just use `.` if that's an option. All backends are assumed to be run from a clean build dir. If you need a subdir for some reason, just hardcode the name instead of exposing to the user

Reply:
Fixed. I removed this option completely, as we used always ``.`` anyways.